openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -0,0 +1,234 @@
+## -*- coding: utf-8 -*-
+
+
+## OpenFisca -- A versatile microsimulation software
+## By: OpenFisca Team <contact@openfisca.fr>
+##
+## Copyright (C) 2011, 2012, 2013, 2014, 2015 OpenFisca Team
+## https://github.com/openfisca
+##
+## This file is part of OpenFisca.
+##
+## OpenFisca is free software; you can redistribute it and/or modify
+## it under the terms of the GNU Affero General Public License as
+## published by the Free Software Foundation, either version 3 of the
+## License, or (at your option) any later version.
+##
+## OpenFisca is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU Affero General Public License for more details.
+##
+## You should have received a copy of the GNU Affero General Public License
+## along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+<%!
+from openfisca_web_site import conf
+%>
+
+
+<%inherit file="/page.mako"/>
+
+<%def name="scripts()" filter="trim">
+<script src="https://code.jquery.com/jquery-3.2.1.min.js"
+    integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+    crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+      //Variables globales
+      var salaire1 = 25000
+      var salaire2 = 15000
+      var revenuDisponible = 0
+
+      window.addEventListener('load', appelALAPI, false);
+
+      // Fonctions
+      function miseAjourDesSalaires(salaire, montant){
+        if (salaire == "salaire1"){
+          salaire1 = Number(montant);
+        }else if (salaire == "salaire2"){
+          salaire2 = Number(montant);
+        };
+        appelALAPI()
+      }
+      function JSONDeLaSituation(){
```

Comment:
Prefer verbs for function names, as a function usually _does_ something.
Also, prefer english to french in the code, unless we are talking about domain concepts (`ImpotSurLaFortune`, `Cotisation`)